docs/releases/1.7.9.txt

```diff
@@ -0,0 +1,10 @@
+==========================
+Django 1.7.9 release notes
+==========================
+
+*Under development*
+
+Bugfixes
+========
+
+* Added renaming of Oracle triggers and sequences when using ``RenameModel`` (:ticket:`24715`).
```

Comment:
please wrap docs at 79 characters and also add this document to docs/releases/index.txt to fix the build failure.